tests/driver_si70xx/main.c

```diff
@@ -44,25 +31,10 @@ int main(void)
 
     puts("SI7021 temperature and humidity sensor test application\n");
 
-    /* enable the sensor if test pin given */
-    if (TEST_PIN_EN != GPIO_UNDEF) {
```

Comment:
Sensor does not have a enable pin, but the development board has one. I've added this to be able to run this program without modifying the source first.